src/CoreWCF.Primitives/src/CoreWCF/Channels/IStreamedMessageEncoder.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+
+namespace CoreWCF.Channels
+{
+    interface IStreamedMessageEncoder
+    {
```

Comment:
Did you add usage for this? I don't think it's needed as it was used in a very narrow scenario in the HttpTransport which didn't get ported across.

Reply:
This has been addressed in f07fd52ed32a91061406fb1366a34d21f874d639.